### PR TITLE
CVSL-341: Add COM Username to LicenceSummary

### PR DESCRIPTION
**Pull request by smcveigh941**

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceServiceTest.kt [hunk 1]
```diff
@@ -705,14 +705,14 @@ class LicenceServiceTest {
       topupSupervisionExpiryDate = LocalDate.of(2021, 10, 22),
       comFirstName = "Stephen",
       comLastName = "Mills",
```
Comment (timharrison-moj) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceServiceTest.kt:707]: Should we remove the first/last names everywhere, and maybe drop the columns from the licence table?  Are they used anywhere now?

**Review by timharrison-moj — APPROVED** [review]
LGTM. One question only.

File: src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceServiceTest.kt (hunk 1 above)
Comment (smcveigh941) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceServiceTest.kt:707]: They're not used. I think there is a task to clean up the DB schema in general

Comment (smcveigh941) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceServiceTest.kt:707]: Will put one in as `tech debt`

**Pull request merged**